tests/test_rat_command.py

```diff
@@ -108,7 +108,7 @@ def test_register_non_callable(self, garbage):
             # print(f"bot={bot}\tchannel={channel}\tsender={sender}")
             return bot, channel, sender
 
-        assert alias.lower() in Commands._REGISTERED_COMMANDS.keys()
+        assert alias.lower() in Commands._registered_commands.keys()
```

Comment:
[suggestion] access to `.keys` is redundant. the contains check for dictionary checks against the keys
```suggestion
        assert alias.lower() in Commands._registered_commands
```